src/lib/common/services/utils/utils.service.ts

```diff
@@ -323,6 +324,21 @@ export class UtilsService {
     });
   }
 
+  handleDeprecatedParameters(config: {[key: string]: any}) {
+    if (config.showMultipleYearsNavigation !== null && config.showMultipleYearsNavigation !== undefined) {
```

Comment:
why not just: 
```
if (config.showMultipleYearsNavigatio) {
  ....
}
```

Reply:
Because I wished to catch it even if the parameter was set to `false`.

Reply:
ok